controllers/errors.controller.js

```diff
@@ -15,5 +15,6 @@ exports.handleCustomErrors = (err, req, res, next) => {
 };
 
 exports.handleInternalServerError = (err, req, res, next) => {
+  console.log(err);
```

Comment:
I'll forgive it, as you're prob using a lot this week, but make sure all console.logs are removed before you host.